python/ray/autoscaler/node_provider.py

```diff
@@ -25,6 +26,7 @@ def load_aws_config():
     "kubernetes": None,
     "docker": None,
     "local_cluster": None,
+    "external": None,  # Import an external module
 }
 
 DEFAULT_CONFIGS = {
```

Comment:
do you have an example yaml where this works and passes the config validation in `create_or_update_cluster`? I might be wrong but this might raise the error at `autoscaler/commands.py` L34.

Reply:
Thanks for spotting this @richardliaw.